### PR TITLE
create dummy array, create header and main, render list

### PR DESCRIPTION
**Pull request by jonatsuji**
first steps of my capstone project (first US) done.

### CODE REVIEW

File: components/GlobalStyles.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+    --color-brown: #564138;
+    
+   }
+   
+   html,
     body {
         padding: 0;
         margin: 0;
         font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+        background-color: #F7F3E3;
```
Comment (MerleMak) [inline components/GlobalStyles.js]:
```suggestion
        background-color: var(--color-beige);
```

File: components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import Image from "next/image";
+import Logo from "../../assets/images/Logo(cragList).png";
+
+export default function Header() {
+  return (
+    <>
+      <StyledHeader>
+        <Image src={Logo} width={100} />
+      </StyledHeader>
+    </>
```
Comment (MerleMak) [inline components/Header/Header.js]:
```suggestion
      <StyledHeader>
        <Image src={Logo} width={100} />
      </StyledHeader>
```
The fragments are only necessary if you have two parent elements inside of your header. :)

File: components/GlobalStyles.js (hunk 1 above)
Comment (AntjeWalter) [inline components/GlobalStyles.js]: könntest du die background-color nicht hier schon mit deiner Variablen angeben?

File: data/lib.js [hunk 3]
```diff
@@ -1,16 +1,420 @@
...
+    grade: "6b+",
+    length: 3,
+    stone: "Sandstone",
+    walkTime: "5-10 Minutes",
+    sector: "Galerie",
+    area: "Avalonia",
+    pos: "51.40295, 7.40755",
+  },
+  {
+    name: "Beißen & Reißen für umme",
```
Comment (MerleMak) [inline data/lib.js:243]: 😂

**Review by AntjeWalter — APPROVED** [review]
LGTM ;)

File: components/Routes/Routes-List.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { routes } from "../../data/lib.js";
+
+export default function RoutesList() {
+  return (
+    <ul>
+      {routes?.map((route) => (
+        <li>
```
Comment (MerleMak) [inline components/Routes/Routes-List.js]: You will need a unique key prop here, so you might want to consider adding an ID to your routes in the data  directory :)

**Review by MerleMak — COMMENTED** [review]
Nice job so far 😊
I left some remarks for you to check out, please also deploy your app on vercel already :)
Keep it up 👍🏼

**Review by MerleMak — APPROVED** [review]
👍🏼

**Pull request merged**